src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java

```diff
@@ -183,14 +215,16 @@ public void run()
 
         if (logger.isDebugEnabled())
         {
-            logger.debug("padding,stream="+ destStream.hashCode()
-                + " padding_bps=" + paddingBps
-                + ",optimal_bps=" + totalOptimalBps
-                + ",current_bps=" + totalCurrentBps
-                + ",target_bps=" + totalTargetBps
-                + ",needed_bps=" + totalNeededBps
-                + ",max_padding_bps=" + maxPaddingBps
-                + ",bwe_bps=" + bweBps);
+            logger.debug("probing" +
```

Comment:
Thanks for raising this, I will revert these changes and re-introduce them in a later PR so that they're compatible with our log collection infrastructure.